test/app_test.dart

```diff
@@ -43,15 +46,47 @@ Future<void> main([List<String>? args]) async {
       defaultRequestTimeout: const Duration(seconds: 2),
       localAppName: 'bar',
       localAppVersion: "1.0.0",
+      metadataPersistenceMode: MetadataPersistenceMode.disabled,
+      logLevel: LogLevel.info,
+      requestTimeout: 1000,
       httpClient: httpClient,
     );
     expect(b.appId, 'myapp1');
     expect(b.baseFilePath.path, Directory.systemTemp.path);
     expect(b.baseUrl, Uri.parse('https://not_re.al'));
     expect(b.defaultRequestTimeout, const Duration(seconds: 2));
+    expect(b.logLevel, LogLevel.info);
+    expect(b.metadataPersistenceMode, MetadataPersistenceMode.disabled);
+    expect(b.requestTimeout, 1000);
     expect(b.httpClient, httpClient);
   });
 
+  test('AppConfiguration can be created and an App could be created', () {
```

Comment:
we have a test `app can be created`. I suggest we name this `AppConfiguration can be created` only since this tests all properties of the config.

Reply:
We already have a test with name "AppConfiguration can be created" but it initializes only the appConfig in Dart. I renamed it to 'AppConfiguration can be initialized'. Then renamed this test to 'AppConfiguration can be created', since it really creates the native app configuration.